bin/sl-pm-install.txt

```diff
@@ -30,6 +30,8 @@ Options:
   --skip-default-install
                       Disable 'npm install' phase of application deployments
                       made on this PM.
+  --in-memory-db      Persist deployment information and metrics in a in-memory
```

Comment:
its file backed not just memory, so can we call this "--json-file-db", to make it clearer?